woocommerce/action-scheduler-admin
# Move Scheduled Actions to the WooCommerce menu

When I wrote this extension all WC Admin enabled features needed to be under the Analytics. Now WC Admin supports adding items to other menus. It isn't an analytics screen and it's more applicable for it to be under WooCommerce.

### Testing

1 - Use this snippet to create a few hundred action if needed
```
add_action( 'init', function() {
    for ( $i = 1; $i < 200; $i++ )
        as_enqueue_async_action( 'as-async-test' . $i );
} );
```
2. run `npm run watch`
3. Go to Analytics -> Scheduled Actions
4. Change the status drop down to `All`
5. A list of actions should load
6. Enable the new navigation experience with
```
add_filter( 'pre_option_woocommerce_navigation_enabled', function() {
return 'yes';
});
add_filter( 'woocommerce_admin_features', function( $features ) {
	$features[] = 'navigation';
	return $features;
});
```
7. Go to the WP dashboard
8. Go to `WooCommerce`
9. `Scheduled Actions` should be in the menu
10. Steps 4 & 5 should produce the same results
